### ISSUE TITLE
new calendar entry is discarded on click within whitespace between cancel and create

### ISSUE THREAD

**Issue by vchrizz** [nextcloud/calendar#1752]
### Steps to reproduce
1. go to calendar
2. create new event
3. click on whitespace between cancel and create buttons

### Expected behaviour
new calendar entry should not vanish or be discarded

### Actual behaviour
new calendar entry vanishes

### Calendar app
**Calendar app version:** 1.7.1

**CalDAV-clients used:** WebInterface

### Client configuration
**Browser:** Google Chrome (v79.0.3945.88) + Mozilla Firefox (v71.0)

**Operating system:** Windows 10

### Server configuration

**Operating system**: Debian 10

**Web server:** Apache2

**Database:** MariaDB

**PHP version:** 7.3

**Nextcloud Version:** 16.07.1

**Updated from an older installed version or fresh install:** fresh install

**List of activated apps:**

```
Enabled:
  - accessibility: 1.2.0
  - activity: 2.9.1
  - bruteforcesettings: 1.4.0
  - calendar: 1.7.1
  - cloud_federation_api: 0.2.0
  - comments: 1.6.0
  - contacts: 3.1.6
  - dav: 1.9.2
  - federatedfilesharing: 1.6.0
  - federation: 1.6.0
  - files: 1.11.0
  - files_external: 1.7.0
  - files_pdfviewer: 1.5.0
  - files_rightclick: 0.15.1
  - files_sharing: 1.8.0
  - files_texteditor: 2.8.0
  - files_trashbin: 1.6.0
  - files_versions: 1.9.0
  - files_videoplayer: 1.5.0
  - firstrunwizard: 2.5.0
  - gallery: 18.3.0
  - logreader: 2.1.0
  - lookup_server_connector: 1.4.0
  - nextcloud_announcements: 1.5.0
  - notifications: 2.4.1
  - oauth2: 1.4.2
  - password_policy: 1.6.0
  - privacy: 1.0.0
  - provisioning_api: 1.6.0
  - recommendations: 0.4.0
  - richdocuments: 3.5.0
  - serverinfo: 1.6.0
  - sharebymail: 1.6.0
  - support: 1.0.0
  - survey_client: 1.4.0
  - systemtags: 1.6.0
  - theming: 1.7.0
  - twofactor_backupcodes: 1.5.0
  - updatenotification: 1.6.0
  - user_ldap: 1.6.0
  - viewer: 1.2.0
  - workflowengine: 1.6.0
Disabled:
  - admin_audit
  - encryption
```

**Nextcloud configuration:**

```
{
    "system": {
        "trusted_proxies": "***REMOVED SENSITIVE VALUE***",
        "overwritehost": "cloud.onetrix.net",
        "overwriteprotocol": "https",
        "overwrite.cli.url": "https:\/\/cloud.onetrix.net\/",
        "htaccess.RewriteBase": "\/",
        "forcessl": "true",
        "forceSSLforSubdomains": "true",
        "overwritewebroot": "\/",
        "overwritecondaddr": "^10\\.5\\.44\\.103$",
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "cloud.onetrix.net",
            "office.onetrix.net",
            "10.5.44.211"
        ],
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "mysql",
        "version": "16.0.7.1",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpmode": "smtp",
        "mail_sendmailmode": "smtp",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mysql.utf8mb4": true,
        "memcache.local": "\\OC\\Memcache\\Redis",
        "memcache.distributed": "\\OC\\Memcache\\Redis",
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "redis": {
            "host": "***REMOVED SENSITIVE VALUE***",
            "port": 0,
            "timeout": 0
        },
        "updater.release.channel": "production",
        "theme": "",
        "log_type": "owncloud",
        "logfile": "\/var\/www\/nextcloud.log",
        "loglevel": 3,
        "logdateformat": "F d, Y H:i:s",
        "ldapIgnoreNamingRules": false,
        "ldapProviderFactory": "OCA\\User_LDAP\\LDAPProviderFactory",
        "auth.bruteforce.protection.enabled": false,
        "mail_smtpport": "25",
        "maintenance": false,
        "updater.secret": "***REMOVED SENSITIVE VALUE***"
    }
}
```

**Comment by FredericLespez** [comment]
I  can confirm this bug.

**Client configuration**
Browser: Mozilla Firefox (v71.0)
Operating system: Debian 10

**Server configuration**
Operating system: Debian 10
Web server: Apache2
Database: PostgreSQL
PHP version: 7.3
Nextcloud Version: 17.0.2

**Comment by FredericLespez** [comment]
As far I can see, this bug is fixed is Calendar 2.0.0.
